resource-service/common/git.go

```diff
@@ -135,12 +140,42 @@ func (g Git) storeDefaultBranchConfig(gitContext common_models.GitContext, err e
 	return nil
 }
 
+func retrieveDefaultBranchFromEnv(env envconfig.EnvConfig) string {
+	if env.DefaultRemoteGitRepositoryBranch == "" {
+		logger.Errorf("Could not determine default remote git repository branch from env variable")
```

Comment:
let's use a different log level (e.g. debug) since it's not really an error if the env var is not set